### PR TITLE
keep quads from being a single color.

### PR DESCRIPTION
**Pull request by willharrison**
added method to color to generate block colors for a quad.  makes sure quads can't be a single color.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +0,0 @@
-# Documentation will be periodically generated and placed in the gh-pages branch
-docs/html
```
Comment (ALSchwalm) [inline .gitignore]: Please don't remove the gitignore. We don't need docs/html in master (as it tends to clutter commits like this)

**Comment by ALSchwalm** [comment]
I don't think there is any problem with the quads being all one color. Yes they will clear as soon as they land, but because creating a group of four of the same color also clears adjacent blocks, it would still be useful to the player.

**Comment by willharrison** [comment]
I wasn't sure if it'd be needed after I did it but figured I would submit it anyways and see what y'all thought.

File: .gitignore (hunk 1 above)
Comment (willharrison) [inline .gitignore]: oops, my bad on that.

**Pull request closed without merging**